README.md

```diff
@@ -0,0 +1,496 @@
+
+All inline comments should start with a `#` and a single space.
+
+The use of inline comments should be limited, because their existence is typically a sign of a code smell.
+
+Do not use inline comments when they state the obvious:
+
+```coffeescript
+  # No
+  x = x + 1 # Increment x
+```
+
+However, inline comments can be useful in certain scenarios:
+
+```coffeescript
+  # Yes
+  x = x + 1 # Compensate for border
+```
+
+<a name="naming_conventions"></a>
+## Naming Conventions
+
+Use `camelCase` (with a leading lowercase character) to name all variables, methods, and object properties.
+
+Use `CamelCase` (with a leading uppercase character) to name all classes. _(This style is also commonly referred to as `PascalCase`, `CamelCaps`, or `CapWords`, among [other alternatives][camel-case-variations].)_
+
+_(The **official** CoffeeScript convention is camelcase, because this simplifies interoperability with JavaScript. For more on this decision, see [here][coffeescript-issue-425].)_
+
+For constants, use all uppercase with underscores:
+
+```coffeescript
+CONSTANT_LIKE_THIS
+```
+
+Methods and variables that are intended to be "private" should begin with a leading underscore:
+
+```coffeescript
+_privateMethod: ->
+```
+
+<a name="functions"></a>
+## Functions
+
+_(These guidelines also apply to the methods of a class.)_
+
+When declaring a function that takes arguments, always use a single space after the closing parenthesis of the arguments list:
+
+```coffeescript
+foo = (arg1, arg2) -> # Yes
+foo = (arg1, arg2)-> # No
+```
+
+Do not use parentheses when declaring functions that take no arguments:
+
+```coffeescript
+bar = -> # Yes
+bar = () -> # No
+```
+
+In cases where method calls are being chained and the code does not fit on a single line, each call should be placed on a separate line and indented by one level (i.e., two spaces), with a leading `.`.
+
+```coffeescript
+[1..3]
+  .map((x) -> x * x)
+  .concat([10..12])
+  .filter((x) -> x < 11)
+  .reduce((x, y) -> x + y)
+```
+
+When calling functions, choose to omit or include parentheses in such a way that optimizes for readability. Keeping in mind that "readability" can be subjective, the following examples demonstrate cases where parentheses have been omitted or included in a manner that the community deems to be optimal:
+
+```coffeescript
+baz 12
+
+brush.ellipse x: 10, y: 20 # Braces can also be omitted or included for readability
+
+foo(4).bar(8)
+
+obj.value(10, 20) / obj.value(20, 10)
+
+print inspect value
+
+new Tag(new Value(a, b), new Arg(c))
+```
+
+You will sometimes see parentheses used to group functions (instead of being used to group function parameters). Examples of using this style (hereafter referred to as the "function grouping style"):
+
+```coffeescript
+($ '#selektor').addClass 'klass'
```

Comment:
No, if you look at the other PR, you'll see I removed that. What you're looking at here is just verbatim what Polarmobile had